src/rpc/common/MetaProcessors.hpp

```diff
@@ -160,22 +163,24 @@ class IfType final {
 };
 
 /**
- * @brief A meta-processor that wraps a validator and produces a custom error in case the wrapped validator fails.
+ * @brief A meta-processor that wraps a validator or modifier and produces a custom error in case the wrapped validator
+ * or modifier fails.
  */
-template <typename SomeRequirement>
+template <typename SomeRequirementOrModifier>
```

Comment:
Maybe better `ReuirementOrModifierType`? Because `Some...` we usually use for a concept names.

And probably better to add a concept here that it is either requirement or modifier.